openoa/utils/met_data_processing.py

```diff
@@ -70,6 +70,27 @@ def circular_mean(x: pd.DataFrame | pd.Series | np.ndarray, axis: int = 0):
     )
 
 
+@series_method(data_cols=["u", "v"])
+def compute_wind_speed(
+    u: pd.Series | str, v: pd.Series | str, data: pd.DataFrame = None
+) -> pd.Series:
+    """Compute the wind speed from the u and v components. This function should only be used for
+    higher resolution wind data, otherwise the resulting wind speed will be lower than expected.
+
+    Args:
+    u(:obj:`pandas.Series` | :obj:`str`): A pandas DataFrame or Series, or a numpy array
```

Comment:
For the "u" and "v" arguments, if a numpy array is allowed, can this be added to the function header and the list of argument types in the docstring too?

Can you also describe the string argument type here, like in the `compute_wind_direction` argument? ("or the name of the column in :py:attr:`data`")